src/kili/exceptions.py

```diff
@@ -1,6 +1,12 @@
 """Exceptions of the package."""
 from typing import Optional
 
+import graphql
+from gql.transport import exceptions
+
+TransportQueryError = exceptions.TransportQueryError
```

Comment:
to document more accurately I would rather write that:
```python
class TransportQueryError(exceptions.TransportQueryError):
     """Raised when ... """
     pass

class GraphQLError(graphql.GraphQLError):
     """"Raised when ... """
    pass
```

Reply:
I can't do that because the exceptions will be raised by graphql and gql libraries.

If I test for `pytest.raises(KiliSdkGraphQLError)`, the test will fail because `isinstance(graphql.GraphQLError(), KiliSdkGraphQLError)` is false